model/base.py

```diff
@@ -16,7 +19,22 @@ def data(self):
     @property
     def df(self):
         """ Get a dataframe of the data """
-        if not self._df:
+        if self._df is None or self._df.empty:
             self._df = pd.DataFrame(self._data).astype(self._dtypes)
 
         return self._df
+    
+    def to_sql(self, connection, **kwargs):
```

Comment:
This is only used for a_contributions.  To be consistent, should we be using this for all the model types?  I assume that we can eventually create something similar called 'to_csv' and switch everything over to that.

Reply:
It's also used for committees and elections, lines 82-83 of main.py
We may not need such a method for to_csv as CSV has no types, no connection, and no complicated concerns like whether to update vs overwrite tables

Reply:
nvm, I see that elections and committees are using the Pandas DataFrame to_sql method.